### PR TITLE
docs: fill doc string of `TimerCallbackValue` and `TimerCallbackStructValue`

### PR DESCRIPTION
**Pull request by rokamu623**
## Pre-review checklist for the PR author

- [ ] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [ ] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/value_objects/callback.py [hunk 1]
```diff
@@ -298,10 +323,28 @@ def __init__(
 
     @property
     def callback_type(self) -> CallbackType:
+        """
+        Get callback type name.
+
+        Returns
+        -------
+        CallbackType
+            callback type
```
Comment (emori-ctc) [inline src/caret_analyze/value_objects/callback.py]:
```suggestion
            Callback type.
```

File: src/caret_analyze/value_objects/callback.py [hunk 2]
```diff
@@ -510,14 +572,41 @@ def __init__(
...
```
Comment (emori-ctc) [inline src/caret_analyze/value_objects/callback.py]:
```suggestion
            Callback type.
```

**Review by emori-ctc — CHANGES_REQUESTED** [review]

File: src/caret_analyze/value_objects/callback.py (hunk 2 above)
Comment (rokamu623) [reply src/caret_analyze/value_objects/callback.py]: Fixed it in [1cec329](https://github.com/rokamu623/CARET_analyze/pull/24/commits/1cec3298f259204f4eefbf122e2412e3903809fb).

File: src/caret_analyze/value_objects/callback.py (hunk 1 above)
Comment (rokamu623) [reply src/caret_analyze/value_objects/callback.py]: Fixed it in [1cec329](https://github.com/rokamu623/CARET_analyze/pull/24/commits/1cec3298f259204f4eefbf122e2412e3903809fb).

**Review by emori-ctc — APPROVED** [review]
LGTM!

**Pull request merged**